test/sagemaker_tests/pytorch/training/integration/local/test_distributed_training.py

```diff
@@ -78,6 +83,8 @@ def test_cpu_nccl(docker_image, sagemaker_local_session, tmpdir):
 
 
 @pytest.mark.skip_gpu
+@pytest.mark.processor("cpu")
+@pytest.mark.model("mnist")
```

Comment:
Looks like this is a multinode test.

pytest.mark.multinode("multinode(2)") ?

Actually, not sure what "train_instance_count=2" means in the context of local tests.

Reply:
Also unsure, hence why i didn't mark it as multinode